src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRentPerDay = 40;
+  const saleForWeekAndMore = 50;
+  const saleForShortTerm = 20;
+  let totalSum = carRentPerDay * days;
+
+  if (days >= 7) {
+    totalSum = days * carRentPerDay - saleForWeekAndMore;
+  } else if (days >= 3) {
```

Comment:
4. [checklist] Prefer `if` with `return` over `if else` to simplify later conditions.